torch/optim/rmsprop.py

```diff
@@ -49,7 +67,20 @@ def step(self, closure=None):
                     grad = grad.add(group['weight_decay'], p.data)
 
                 square_avg.mul_(alpha).addcmul_(1 - alpha, grad, grad)
-                avg = square_avg.sqrt().add_(group['eps'])
-                p.data.addcdiv_(-group['lr'], grad, avg)
+
+                avg = None
```

Comment:
No need to set it to `None`. Variables declared inside the `if` blocks are visible outside too.